File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/CheckForUpdate.java
package me.grishka.houseclub.api.methods;

import java.util.HashMap;

import me.grishka.houseclub.api.BaseResponse;
import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class CheckForUpdate extends ClubhouseAPIRequest<BaseResponse> {
    public CheckForUpdate() {
        super("GET", "check_for_update", BaseResponse.class);
        queryParams = new HashMap<>();
        queryParams.put("is_testflight", "0");
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/CheckForUpdate.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.BaseResponse
import me.grishka.houseclub.api.ClubhouseAPIRequest

class CheckForUpdate : ClubhouseAPIRequest<BaseResponse?>("GET", "check_for_update", BaseResponse::class.java) {
    init {
        queryParams = hashMapOf("is_testflight" to "0")
    }
}